packages/server/src/fhir/repo.ts

```diff
@@ -972,12 +973,43 @@ export class Repository extends BaseRepository implements FhirRepository {
         } as BundleEntry)
     );
 
-    if (searchRequest.include) {
-      entries.push(...(await this.getSearchIncludeEntries(searchRequest.include, resources)));
-    }
+    if (searchRequest.include || searchRequest.revInclude) {
```

Comment:
I imagine that Sonar will complain about code complexity here.  This method has had a lot of churn, and a lot of refactoring to keep complexity down.